lib/screens/authentication/verification/interactor/viewmodels/verification_state.dart

```diff
@@ -69,3 +71,17 @@ class VerificationState extends Equatable {
     );
   }
 }
+
+extension LocalizedPasCodeTitle on PassCodeTitle {
```

Comment:
A wee typo here 🤓
LocalizedPasCodeTitle -> LocalizedPassCodeTitle

Reply:
btw Its passCode or Passcode ??

Reply:
Good call, @RaulUrtecho! It is [Passcode](https://dictionary.cambridge.org/us/dictionary/english/passcode), same as Password 🤓